concordium-cis2/src/lib.rs

```diff
@@ -513,8 +499,7 @@ macro_rules! token_amount_wrapper {
         impl IsTokenAmount for $name {}
 
         impl schema::SchemaType for $name {
-            // TODO Fix the schema when supporting LEB128
-            fn get_type() -> schema::Type { schema::Type::Array($size, Box::new(schema::Type::U8)) }
+            fn get_type() -> schema::Type { schema::Type::ULeb128(37) }
```

Comment:
Please explain the 37 constant in the comments.

Reply:
I have added a comment